nav2_theta_star_planner/README.md

```diff
@@ -1,52 +1,51 @@
 
-## Features 
+##Features 
 - Uses the costs from the costmap to penalise high cost regions
 - Is well suited for smaller robots of the omni-directional and differential drive kind
 - As it considers the costmap traversal cost during execution it tends to smoothen the paths automatically, thus mitigating the need to smoothen the path. (The presence of sharp turns depends on the resolution of the map, and it decreases as the map resolution increases.)
 - Allows to control the path behavior to either be any angle directed or to be in the middle of the spaces
 - The planner uses A\* search along with line of sight checks to form any-angle paths thus avoiding zig-zag paths that may be present in the usual implementation of A\*  
 
-## Metrics
-For the below example the planner took ~44ms to compute the path of 87.5m - 
-
-![example](img/00-37.png)
+##Metrics
+For the below example the planner took ~46ms (averaged value) to compute the path of 87.5m - 
+![example.png](/img/00-37.png)
 
 The parameters were set to - `w_euc_cost: 1.0`, `w_traversal_cost: 5.0`, `w_heuristic_cost: 1.0` and the `global_costmap`'s `inflation_layer` parameters are set as - `cost_scaling_factor:5.0`, `inflation_radius: 5.5`
 
-## Cost Function Details
-### Symbols and their meanings
-**g(a)** - cost function cost for the node 'a'
+##Cost Function Details
+###Symbols and their meanings
+**g(a)** - cost function cost for the init_rclcpp 'a'
 
-**h(a)** - heuristic function cost for the node 'a'
+**h(a)** - heuristic function cost for the init_rclcpp 'a'
 
-**f(a)** - total cost (g(a) + h(a)) for the node 'a'
+**f(a)** - total cost (g(a) + h(a)) for the init_rclcpp 'a'
 
 **LETHAL_COST** - a value of the costmap traversal cost that inscribes an obstacle with
 respect to a function, value = 253
 
-**curr** - represents the node whose neighbours are being added to the list
+**curr** - represents the init_rclcpp whose neighbours are being added to the list
 
 **neigh** - one of the neighboring nodes of curr
 
-**par** - parent node of curr
+**par** - parent init_rclcpp of curr
 
-**euc_cost(a,b)** - euclidean distance between the node type 'a' and type 'b'
+**euc_cost(a,b)** - euclidean distance between the init_rclcpp type 'a' and type 'b'
 
-**costmap_cost(a,b)** - the costmap traversal cost (ranges from 0 - 252, 254 = unknown value) between the node type 'a' and type 'b'
+**costmap_cost(a,b)** - the costmap traversal cost (ranges from 0 - 252, 254 = unknown value) between the init_rclcpp type 'a' and type 'b'
 
 ### Cost function
 ```
 g(neigh) = g(curr) + w_euc_cost*euc_cost(curr, neigh) + w_traversal_cost*(costmap_cost(curr,neigh)/LETHAL_COST)^2
-h(neigh) = w_heuristic_cost * euc_cost(neigh, goal)
+h(neigh) = w_heuristic_cost * euc_cost(neigh, par)
```

Comment:
Why not `goal`?

Reply:
It is goal, I think I changed the wrong line, let me fox that